### ISSUE TITLE
Docs incorrectly state default (supervised) -loss is "ns"; in fact it's "softmax"

### ISSUE THREAD

**Issue by steveleward** [facebookresearch/fastText#90]
This could be fixed by changing line 41 of "args.cc" from:

loss = loss_name::softmax;

to: 

loss = loss_name::ns;

Note also: "-loss" is the only hyperparameter whose value is not streamed into the doc string (line 140 of "args.cc"), which is perhaps how the error occurred.

**Comment by EdouardGrave** [comment]
You are correct, the `printHelp` function should be updated. We are aware of this issue and will fix it.

**Comment by steveleward** [comment]
Thanks Edouard for your very quick response, and for the pending fix. It was a "gotcha" for me, because at first I thought fasttext was much slower than advertised, and it took me a while to realise the problem was in the docs. Happily, it's _super_ quick when loss is negative sampling. Best, Steve

**Comment by EdouardGrave** [comment]
Hi Steve, you might also want to try the hierarchical softmax (using `-loss hs`), as it is (much) faster than negative sampling at test time. Best, Edouard

**Comment by EdouardGrave** [comment]
This should be fixed now.

**Comment by steveleward** [comment]
Hi Edouard, thanks so much for fixing this so quickly, and for giving me the hint about hierarchical softmax. I'll try it out! Best, Steve
